connector/protobuf/src/test/scala/org/apache/spark/sql/protobuf/ProtobufSerdeSuite.scala

```diff
@@ -126,14 +122,6 @@ class ProtobufSerdeSuite extends SharedSparkSession with ProtobufTestBase {
       params = Map(
         "protobufType" -> "FieldMissingInProto",
         "toType" -> toSQLType(CATALYST_STRUCT)))
-
-    assertFailedConversionMessage(protoFile,
```

Comment:
This test variation does not make sense since `nonnullCatalyst.toSql` produces different result from `CATALYST_STRUCT.toSql`. Non-nullable fields are tested in "Fail to convert with missing Catalyst fields"